src/Utility/TypeExtensions.cs

```diff
@@ -99,6 +90,21 @@ internal static object ToObject(this TypedData data)
             }
         }
 
+        private static object GetRawBody(TypedData rpcHttpBody)
+        {
+            switch (rpcHttpBody.DataCase)
+            {
+                case TypedData.DataOneofCase.String:
+                    return rpcHttpBody.String;
+
+                case TypedData.DataOneofCase.Bytes:
+                    return Encoding.UTF8.GetString(rpcHttpBody.Bytes.ToByteArray());
+
+                default:
+                    return rpcHttpBody.ToObject();
```

Comment:
For the default case in which ```rpcHttpBody``` is neither a string nor bytes, then both ```httpRequestContext.Body``` and ```httpRequestContext.RawBody```  will be rpcHttp.Body.ToObject(). Is this intentional?

Reply:
As far as I understand, the host will be sending us either `TypedData.DataOneofCase.String` or `TypedData.DataOneofCase.Bytes` now, so the default branch will not be executed. However, we need to come up with a safe default behavior if the host sends anything else in the future. Invoking `ToObject()` seems to be a reasonable default to me, but I'm open to other suggestions.

Also, don't pay too much attention to the resulting `httpRequestContext.Body` now. This PR focuses on `RawBody` only, and the fix for https://github.com/Azure/azure-functions-powershell-worker/issues/241 is also coming soon, in a separate PR.

Reply:
I thought about it a bit more. I was thinking that we should keep the same behavior as other workers. However, this has not been implemented in other workers.

Would it be possible to add a TODO there to check how other workers will handle this? Other than that, LGTM

Reply:
This is in fact implemented in the NodeJS worker (see [fromRpcHttpBody](https://github.com/Azure/azure-functions-nodejs-worker/blob/d6a0c97ef7b0142a5c737c5b80d939da060bb189/src/converters/RpcHttpConverters.ts#L40)), and it seems to be doing the same.

Reply:
Great. Thanks for checking.